plugins/check_port/lang/en.js

```diff
@@ -13,5 +13,7 @@
  				  "Port is closed",
  				  "Port is open"
  				  ];
+ theUILang.clickReveal	= "Click to Reveal";
+ theUILang.clickHide		= "Click to Hide";
 
-thePlugins.get("check_port").langLoaded();
\ No newline at end of file
+thePlugins.get("check_port").langLoaded();
```

Comment:
Could you submit a separate pull request for this change? And reference this pull request?